website/docs/components/app-header/partials/code/how-to-use.md

```diff
@@ -0,0 +1,177 @@
+!!! Warning
+
+The `AppHeader` is being rolled out in a phased approach. In the phase one release, we will assist consumers in adoption spikes to optimize it to their needs. This unique approach is due to how the `AppHeader` tightly integrates into the release plan for the “Enterprise navigation” which includes the `SideNav` and `AppFrame` components.
```

Comment:
I think this banner is clear.

Just FYI if you want you can add a "title" to it to make it stand out even more (see [examples here](https://helios.hashicorp.design/testing/components/banner)) by using a "strong" paragraph on its own (eg. `\n**My title**\n`)